applications/FluidDynamicsApplication/python_scripts/navier_stokes_ale_fluid_solver.py

```diff
@@ -15,6 +15,74 @@ def CreateSolver(model, solver_settings, parallelism):
+                time_scheme_fluid = fluid_settings["time_scheme"].GetString()
+                alpha_fluid = fluid_settings["alpha"].GetDouble()
+                if mesh_vel_calc_settings.Has("time_scheme"):
+                    time_scheme_mesh_vel = mesh_vel_calc_settings["time_scheme"].GetString()
+                    if time_scheme_fluid != time_scheme_mesh_vel and self.is_printing_rank:
+                        info_msg  = '"time_scheme" of the fluid (' + time_scheme_fluid
+                        info_msg += ') is different from the\n"time_scheme" used for the '
+                        info_msg += 'calculation of the mesh-velocity (' + time_scheme_mesh_vel + ')'
+                        KratosMultiphysics.Logger.PrintInfo("::[ALEFluidSolver]::", info_msg)
+                else:
+                    mesh_vel_calc_settings.AddValue("time_scheme", fluid_settings["time_scheme"])
+                    if self.is_printing_rank:
+                        info_msg  = 'setting "time_scheme" of the mesh-solver for the\ncalculation of the '
+                        info_msg += 'mesh-velocity to "' + time_scheme_fluid + '" to be consistent with the\n'
+                        info_msg += '"time_scheme" of the fluid'
+                        KratosMultiphysics.Logger.PrintInfo("::[ALEFluidSolver]::", info_msg)
+                if not mesh_vel_calc_settings["time_scheme"].GetString().startswith("bdf"):
+                    if mesh_vel_calc_settings.Has("alpha_m"):
+                        alpha_mesh_vel = mesh_vel_calc_settings["alpha_m"].GetDouble()
+                        if abs(alpha_fluid-alpha_mesh_vel) > 1e-12 and self.is_printing_rank:
+                            info_msg  = '"alpha" of the fluid (' + str(alpha_fluid)
+                            info_msg += ') is different from the\n"alpha_m" used for the '
+                            info_msg += 'calculation of the mesh-velocity (' + str(alpha_mesh_vel) + ')'
+                            KratosMultiphysics.Logger.PrintInfo("::[ALEFluidSolver]::", info_msg)
+                    else:
+                        mesh_vel_calc_settings.AddValue("alpha_m", fluid_settings["alpha"])
+                        if self.is_printing_rank:
+                            info_msg  = 'setting "alpha_m" of the mesh-solver for the\ncalculation of the '
+                            info_msg += 'mesh-velocity to "' + str(alpha_fluid) + '" to be consistent\nwith '
+                            info_msg += '"alpha" of the fluid'
+                            KratosMultiphysics.Logger.PrintInfo("::[ALEFluidSolver]::", info_msg)
+
+        elif fluid_solver_type == "fractional_step" or fluid_solver_type == "FractionalStep":
+            # currently fractional step always uses BDF2
+            if mesh_vel_calc_settings.Has("time_scheme"):
+                time_scheme_mesh_vel = mesh_vel_calc_settings["time_scheme"].GetString()
+                if time_scheme_mesh_vel != "bdf2" and self.is_printing_rank:
```

Comment:
Try to avoid hard-coding the bdf2 scheme here. It creates maintenance problems if someone later changes the fractional step scheme. Again, I don't know if this is possible with the current fluid implementation.